tests/lua/machine-bind.lua

```diff
@@ -1497,4 +1497,206 @@ test_util.make_do_test(build_machine, machine_type, {
     assert(log.accesses[7].written == leaf_data)
 end)
 
+-- helper function to load a step log file into a table
+local function read_step_log_file(filename)
+    local file <close> = io.open(filename, "rb")
```

Comment:
Wrap `io.open` with `assert()`, here in all new `io.open` below, otherwise my Lua linter will complain.